rust/shared/warp_bindgen_macro/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "warp_bindgen_macro"
+version = "0.1.0"
+edition = "2021"
+description = "Warp WASM contract macro for rust contracts"
+license = "MIT"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+quote = "1.0"
+syn = { version = "1.0", features = ["extra-traits"] }
+wasm-bindgen = { version = "0.2.79", features = ["serde-serialize"] }
+wasm-bindgen-futures = { version = "0.4.29" }
```

Comment:
new lines..

Reply:
There is no way to brake so-called 'inline tables' in TOML world into multiple lines. Also this format is being used among all the tutorials.